github/build.sbt

```diff
@@ -0,0 +1,2 @@
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.7.1")
```

Comment:
We have to be careful with our versioning strategy here. I'm rolling this back from 1.8 because otherwise I can't use it with http4s-dom which is currently blocked by upstream issues with 1.8, see https://github.com/http4s/http4s-dom/pull/60#issuecomment-1001079167.